test/e2e/tilesets-import.test.js

```diff
@@ -272,7 +271,6 @@ test.skip('POST /tilesets/import fails when providing invalid mbtiles, no tilese
     payload: { filePath: badMbTilesPath },
   })
 
-  // This is currently 500, but should probably be 400?
   t.equal(importResponse.statusCode, 400)
```

Comment:
Maybe we should check here that the line 142 in `imports.ts` is actually working, e.g. is this being reported as an mbtiles error?